packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -5,14 +5,8 @@ import {
   ResultTemplatesHelpers,
 } from '@coveo/headless';
 import {MapProp} from '../../utils/props-utils';
-import {containsSection} from '../../utils/result-section-utils';
-import {ResultDisplayImageSize} from '../atomic-result/atomic-result-display-options';
 
-export interface TemplateContent {
-  innerHTML: string;
-  usesSections: boolean;
-  imageSize?: ResultDisplayImageSize;
-}
+export type TemplateContent = DocumentFragment;
 
```

Comment:
This is one of the most important parts of this change. We no longer need all these attributes  with the template.